### DISCUSSION TITLE
Update to git instead of ZIP install method

### DISCUSSION THREAD

**Discussion by Alkarex** [FreshRSS/FreshRSS#6769] [Q&A] [answered]
If available, installing and updating FreshRSS via git is more robust than via the ZIP method.
Both methods allow updates via the Web interface, as long as the file access rights are sufficient.

**Comment by Alkarex** [comment, marked as answer]
If you initially installed FreshRSS via ZIP, here is a series of command to update to the git method, targetting the latest FreshRSS release:

```sh
cd FreshRSS/
git init
git remote add origin https://github.com/FreshRSS/FreshRSS.git
git fetch
git checkout -f -b latest origin/latest
cli/access-permissions.sh
# If you want to allow updates through the Web UI (at the cost of decreasing a bit the security):
sudo chown www-data:www-data -R .
```

You can then follow the instructions for git updates:
* https://freshrss.github.io/FreshRSS/en/admins/04_Updating.html#using-git
* https://freshrss.github.io/FreshRSS/en/admins/07_LinuxUpdate.html

**Reply by Frenzie** [reply]
I'd probably make a backup copy first just in case.

**Reply by scharp19** [reply]
> Если вы изначально установили FreshRSS через ZIP-архив, вот несколько команд для перехода на метод git с использованием последней версии FreshRSS:
> 
> ```shell
> cd FreshRSS/
> git init
> git remote add origin https://github.com/FreshRSS/FreshRSS.git
> git fetch
> git checkout -f -b latest origin/latest
> cli/access-permissions.sh
> # Если вы хотите разрешить обновления через веб-интерфейс (за счет некоторого снижения уровня безопасности):
> sudo chown www-data:www-data -R .
> ```
> 
> Затем вы можете следовать инструкциям по обновлению git:
> 
> * https://freshrss.github.io/FreshRSS/en/admins/04_Updating.html#using-git
> * https://freshrss.github.io/FreshRSS/en/admins/07_LinuxUpdate.html

Thanks, I'd rather wait for you to prepare the zip archive. It's too difficult for me to update via GIT)

**Comment by scharp19** [comment]
Thanks, I'd rather wait for you to prepare the zip archive. It's too difficult for me to update via GIT)
